GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 
+//swiftlint:disable file_length
```

Comment:
Not sure... looks like cheating.

Reply:
Absolutely, but I prefer to do this instead of splitting the file or increasing the file length. If this file grows much more I will consider to split into files, since now it is readable IMO.